assets/static/images/bell--inactive.svg

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="34px" height="34px" viewBox="0 0 34 34" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 49.1 (51147) - http://www.bohemiancoding.com/sketch -->
+    <title>Fill 1</title>
+    <desc>Created with Sketch.</desc>
+    <defs>
+        <linearGradient x1="0%" y1="50%" x2="100%" y2="50%" id="linearGradient-1">
+            <stop stop-color="#D0D3EA" offset="0%"></stop>
+            <stop stop-color="#D0D3EA" offset="100%"></stop>
+        </linearGradient>
+    </defs>
+    <g id="10books" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Library---main" transform="translate(-1732.000000, -409.000000)" fill="url(#linearGradient-1)">
+            <g id="Group-32" transform="translate(110.000000, 385.000000)">
+                <g id="1">
+                    <g id="Icon-notify" transform="translate(1622.000000, 24.000000)">
+                        <path d="M24.4674698,21.9753333 L24.4669031,21.9753333 L24.4669031,21.97522 C24.4669031,22.7923533 23.804692,22.8461867 22.9881199,22.8461867 L11.0114267,22.8461867 C10.1945146,22.8461867 9.53287022,22.7923533 9.53287022,21.97522 L9.53287022,21.83446 C9.53287022,21.2467133 9.87808585,20.7436267 10.3739225,20.5048333 L10.843239,16.4528267 C10.843239,13.4865533 12.9410529,11.01056 15.7338316,10.4265533 L15.7338316,8.85348667 C15.7338316,8.15422 16.300842,7.5871 17.0004533,7.5871 C17.6997247,7.5871 18.2667351,8.15422 18.2667351,8.85348667 L18.2667351,10.4266667 C21.0596271,11.01056 23.1573277,13.4863267 23.1573277,16.45294 L23.6267575,20.5055133 C24.1225942,20.7446467 24.4674698,21.2468267 24.4674698,21.8345733 L24.4674698,21.9753333 Z M16.9998867,26.4011133 C15.7797319,26.4011133 14.7899853,25.4121667 14.7899853,24.1912267 L19.2094481,24.1912267 C19.2096747,25.4121667 18.2200415,26.4011133 16.9998867,26.4011133 Z M16.9995467,0 C7.61117741,0 0,7.61158 0,16.99966 C0,26.38774 7.61117741,34 16.9995467,34 C26.3880293,34 34,26.3881933 34,16.99966 C34,7.61112667 26.3880293,0 16.9995467,0 Z" id="Fill-1"></path>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
use SVGOMG (https://jakearchibald.github.io/svgomg/) to make SVG nicer and get rid of unused info in svg file :)